src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/EndedReferralDTO.kt

```diff
@@ -21,6 +24,9 @@ class EndedReferralDTO(
         referenceNumber = referral.referenceNumber!!,
         assignedTo = referral.assignedTo?.let { AuthUserDTO.from(it) },
         referral = DraftReferralDTO.from(referral),
+        cancellationReason = referral.cancellationReason!!.description,
```

Comment:
please can we leave these as nullable for now? i'm not sure that ended referrals will always have cancellation reasons and comments